db/models.py

```diff
@@ -14,3 +14,39 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
```

Comment:
There is a code efficiency issue here. You are overusing f-string. Since `self.first_name` and `self.last_name` are both strings, you can use the '+' operator to concatenate them. Corrected code: `return self.first_name + ' ' + self.last_name`.